### PR TITLE
Improve README.md, prepare for 0.1.0 🚀

### PR DESCRIPTION
**Pull request by p4checo**
- Improved `README.md`

- Cleaned up `.podspec`

- Configured `.travis.yml` to deploy to GitHub Releases and push to CocoaPods trunk

### CODE REVIEW

**Comment by p4checo** [comment]
I didn't 😅 

But the link you shared doesn't seem to be what we want 🤔 Got any other example?

**Comment by portellaa** [comment]
Yes, this is for blogs, but i was sharing because of the authentication part that needs to be done, which i think is covered by that blog, or not 🤔 

Regarding the submission to cocoapods, it's an extra command travis.yml with `pod trunk push`

**Comment by portellaa** [comment]
👉 https://fuller.li/posts/automated-cocoapods-releases-with-ci/ 😅 this is better 😂

File: README.md [hunk 1]
```diff
@@ -1,21 +1,104 @@
...
+[![CocoaPods](https://img.shields.io/cocoapods/v/Alicerce.svg)](https://cocoapods.org/)
+[![SwiftPM](https://img.shields.io/badge/SwiftPM-compatible-orange.svg)](#swift-package-manager)
 [![Build Status](https://travis-ci.org/Mindera/Alicerce.svg?branch=master)](https://travis-ci.org/Mindera/Alicerce)
 [![codecov](https://codecov.io/gh/Mindera/Alicerce/branch/master/graph/badge.svg)](https://codecov.io/gh/Mindera/Alicerce)
 
+## What is it? 🤔
+
+Ever felt that you keep repeating yourself every time you start a new project? That you would like to have all those useful utils and helpers you love already available? We felt that way too! Thus, Alicerce was born. 🏗
+
+Alicerce is a framework that aims to serve as a starting point for iOS applications, by providing the foundations for many of the common functionalities a modern application requires, as well as be a repository for those small utils and helpers that make our life easier.
```
Comment (mschuetz72) [inline README.md]: is it "application requires", or "application requirements"? Now I got confused myself 🤣

Comment (mschuetz72) [reply README.md]: Ok, read it again and of course it makes sense the way it is...

File: README.md [hunk 2]
```diff
@@ -1,21 +1,104 @@
...
+
+1. Add this repository as a [submodule][].
+1. Drag Alicerce.xcodeproj into your project or workspace.
+1. Link your target against Alicerce.framework.
+1. If linking against an Application target, ensure the framework gets copied into the bundle. If linking against a Framework target, 
+the application linking to it should also include Alicerce.
+
+## Contributing 🙌
+
+See [CONTRIBUTING](https://github.com/Mindera/fastlane-plugin-cosigner/blob/master/CONTRIBUTING.md).
```
[the remark this answers is not in the document]
Comment (p4checo) [reply README.md]: nice catch! Will fix 😉

**Pull request merged**